Failed to compile after the recent changes in the apm-rum-js agent

```
22:39:55  Failed to compile.
22:39:55  
22:39:55  Module not found: Error: Can't resolve '@elastic/apm-rum' in '/app/src'
22:39:55  
```

See [build](https://apm-ci.elastic.co/job/apm-agent-nodejs/job/opbeans-frontend-mbp/job/master/8/console) for further details

--- a/Dockerfile
+++ b/Dockerfile
@@ -6,7 +6,8 @@ ENV ELASTIC_APM_JS_BASE_SERVICE_NAME=opbeans-react
 WORKDIR /app
 COPY . /app
 
-RUN yarn install \
+# Need it to enable preinstall scripts in remote host
+RUN npm install --unsafe-perm \
     && npm run-script build
 
 CMD ["npm", "run-script", "start"]

--- a/lerna.json
+++ b/lerna.json
@@ -1,0 +1,6 @@
+{
+  "packages": [
+    "elastic-apm-rum/packages/*"
+  ],
+  "version": "0.0.0"
+}

--- a/package.json
+++ b/package.json
@@ -61,7 +61,7 @@
   },
   "scripts": {
     "preinstall": "if ! [ -d elastic-apm-rum ]; then git clone https://github.com/elastic/apm-agent-js-base elastic-apm-rum; fi",
-    "postinstall": "npm install --prefix ./elastic-apm-rum && npx lerna run build --stream",
+    "postinstall": "npm install --prefix ./elastic-apm-rum --only dev --no-package-lock && npx lerna run build --stream",
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom"

--- a/package.json
+++ b/package.json
@@ -60,8 +60,8 @@
     "whatwg-fetch": "1.0.0"
   },
   "scripts": {
-    "preinstall": "if ! [ -d elastic-apm-rum ]; then git clone https://github.com/elastic/apm-agent-js-base elastic-apm-rum; fi",
-    "postinstall": "npm install --prefix ./elastic-apm-rum --only dev --no-package-lock && npx lerna run build --stream",
+    "preinstall": "if ! [ -d elastic-apm-rum ]; then git clone https://github.com/elastic/apm-agent-rum-js elastic-apm-rum --depth 1; fi",
+    "postinstall": "npm install --prefix ./elastic-apm-rum --dev --prod --no-package-lock && npx lerna run build --stream",
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom"

--- a/package.json
+++ b/package.json
@@ -36,6 +36,7 @@
     "http-proxy-middleware": "0.17.2",
     "jest": "17.0.2",
     "json-loader": "0.5.4",
+    "lerna": "^3.16.4",
     "moment": "^2.17.1",
     "object-assign": "4.1.0",
     "path-exists": "2.1.0",
@@ -60,6 +61,7 @@
   },
   "scripts": {
     "preinstall": "if ! [ -d elastic-apm-rum ]; then git clone https://github.com/elastic/apm-agent-js-base elastic-apm-rum; fi",
+    "postinstall": "npm install --prefix ./elastic-apm-rum && npx lerna run build --stream",
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom"

--- a/package.json
+++ b/package.json
@@ -61,7 +61,7 @@
   },
   "scripts": {
     "preinstall": "if ! [ -d elastic-apm-rum ]; then git clone https://github.com/elastic/apm-agent-rum-js elastic-apm-rum --depth 1; fi",
-    "postinstall": "npm install --prefix ./elastic-apm-rum --dev --prod --no-package-lock && npx lerna run build --stream",
+    "postinstall": "npm install --prefix ./elastic-apm-rum --dev --prod && npx lerna run build --stream",
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom"

